gazebo/rendering/Distortion.cc

```diff
@@ -182,13 +182,18 @@ void Distortion::SetCamera(CameraPtr _camera)
   const double fov = _camera->ImageHeight() > _camera->ImageWidth() ?
       _camera->VFOV().Radian() : _camera->HFOV().Radian();
   const double focalLength = texSide/(2*tan(fov/2));
-  this->dataPtr->distortionTexWidth = texSide - 1;
-  this->dataPtr->distortionTexHeight = texSide - 1;
+  this->dataPtr->distortionTexWidth = texSide;
+  this->dataPtr->distortionTexHeight = texSide;
   unsigned int imageSize =
       this->dataPtr->distortionTexWidth * this->dataPtr->distortionTexHeight;
   double colStepSize = 1.0 / this->dataPtr->distortionTexWidth;
   double rowStepSize = 1.0 / this->dataPtr->distortionTexHeight;
 
+  // Half step-size vector to add to the value being placed in distortion map.
+  // Necessary for compositor to correctly interpolate pixel values.
+  const auto halfStepVec =
```

Comment:
"step" is pretty generic. I would call this "halfTexelSize".

Reply:
Changed it, thanks